### PR TITLE
Doc/eventhandling

### PR DESCRIPTION
**Pull request by Rboin**
Rebase and merge after #53.
- [x] Technical documentation written
- [x] Technical documentation reviewed

### CODE REVIEW

File: docs/software_architecture/chapters/control-workers.tex [hunk 1]
```diff
@@ -1,26 +1,26 @@
 \section{Controlling workers}
 
-One of the things the player can do is order workers to do a specific task. 
-This is done by selecting one or more workers followed by right clicking 
-somewhere. For example, the player can right click on the ground, which will 
-cause the selected workers to move there. The player can also order workers 
+One of the things the player can do is order workers to do a specific task.
+This is done by selecting one or more workers followed by right clicking
+somewhere. For example, the player can right click on the ground, which will
```
Comment (MarkVlamPersie) [inline docs/software_architecture/chapters/control-workers.tex]: the player can right click **on** the ground

im not sure if the word 'on' is appropriate here.

File: docs/software_architecture/chapters/control-workers.tex [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 to gather resources such as wood.
 
-Since the player can select any kind of worker and order it to do many 
-different things, a design pattern was needed to ensure that the code 
-remained readable and extensible. The strategy pattern was chosen because of 
-the fact that it is very extensible. 
+Since the player can select any kind of worker and order it to do many
+different things, a design pattern was needed to ensure that the code
+remained readable and extensible. The strategy pattern was chosen because of
+the fact that it is very extensible.
```
Comment (MarkVlamPersie) [inline docs/software_architecture/chapters/control-workers.tex]: All SOLID design patterns are extensible. That is not really a specific reason to choose for the strategy pattern.

File: docs/software_architecture/chapters/control-workers.tex (hunk 1 above)
Comment (Rboin) [inline docs/software_architecture/chapters/control-workers.tex]: Not really relevant to my documentation, rebased the docs branch on top of mine, but did not touch the file.

File: docs/software_architecture/chapters/control-workers.tex (hunk 2 above)
Comment (Rboin) [inline docs/software_architecture/chapters/control-workers.tex]: Not really relevant to my documentation, rebased the docs branch on top of mine, but did not touch the file.

**Pull request merged**